<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionToolForm</class>
 <widget class="QWidget" name="OptionToolForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>ビューア1(&amp;1):</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_ViewerName1</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_ViewerName1">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_SelectViewer1">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>ビューア1は、アプリケーションに関連づけられていないファイルを「開く」で開くときにも使用します。また、「ファイル一覧情報の詳細表示」でも使用します。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ビューア2(&amp;2):</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_ViewerName2</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit_ViewerName2">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_SelectViewer2">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ビューア3(&amp;3):</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_ViewerName3</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEdit_ViewerName3">
       <property name="maxLength">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_SelectViewer3">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="text">
      <string>ビューア2、3はマウス右ボタンクリックで使用できます。</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>247</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton_SelectViewer1</sender>
   <signal>clicked()</signal>
   <receiver>OptionToolForm</receiver>
   <slot>onClick_toolButton_SelectViewer1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_SelectViewer2</sender>
   <signal>clicked()</signal>
   <receiver>OptionToolForm</receiver>
   <slot>onClick_toolButton_SelectViewer2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_SelectViewer3</sender>
   <signal>clicked()</signal>
   <receiver>OptionToolForm</receiver>
   <slot>onClick_toolButton_SelectViewer3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClick_toolButton_SelectViewer1()</slot>
  <slot>onClick_toolButton_SelectViewer2()</slot>
  <slot>onClick_toolButton_SelectViewer3()</slot>
 </slots>
</ui>
